python/manylinux1/Dockerfile-x86_64_base

```diff
@@ -17,7 +17,7 @@
 
 # See https://quay.io/repository/pypa/manylinux1_x86_64?tab=history
 # to update base image.
-FROM quay.io/pypa/manylinux1_x86_64@sha256:e7ed385da2363ea796385bb106a98364f7a8a658a5e8dbf1b7e4c71f2f299251
+FROM quay.io/pypa/manylinux1_x86_64:latest
```

Comment:
We should pin the specified version to avoid unexpected failure.

Reply:
looks like only latest manifests are hosted at https://quay.io/repository/pypa/manylinux1_x86_64?tab=tags

Reply:
We can find SHA256 at https://quay.io/repository/pypa/manylinux1_x86_64?tab=history .
This URL is described in the above comment.

Reply:
the manifest will be removed once 'latest' tag is updated

Reply:
Oh...
It seems that there is the `2020-01-31-d8fa357` tag. Can we use this?

Reply:
no, both 'latest' and '2020-01-31-d8fa357' point to same manifest

Reply:
@pprudhvi What @kou means is to avoid using a moving tag such as `latest`, you can use the fixed `2020-01-31-d8fa357`.

Reply:
done but I am not sure if the manifest which '2020-01-31-d8fa357' (and 'latest') point to will be present once 'latest' tag is moved.